pkg/apis/config/v1beta1/zz_generated.defaults.go

```diff
@@ -28,29 +28,5 @@ import (
 // Public to allow building arbitrary schemes.
 // All generated defaulters are covering - they call all nested defaulters.
 func RegisterDefaults(scheme *runtime.Scheme) error {
-	scheme.AddTypeDefaultingFunc(&CapacitySchedulingArgs{}, func(obj interface{}) { SetObjectDefaultsCapacitySchedulingArgs(obj.(*CapacitySchedulingArgs)) })
```

Comment:
I guess it's caused by autogen script. But let's revert changes in this file anyhow.

Reply:
Yes, copied back.